### PR TITLE
[5/11] Headers for jobs

### PR DESCRIPTION
**Pull request by philipnrmn**
---
~~⚠️   _This branch depends on features that will be introduced with #1482_~~ Merged!

---

This is a WIP PR which adds headers to the jobs section. 

The TaskDetail section is still missing, this is awkward because there are several valid routes for the view via services, jobs, and apparently network also. The answer to this is probably to write individual pages for each route which share the common component.

### CODE REVIEW

File: src/js/pages/__tests__/JobsTab-test.js [hunk 1]
```diff
@@ -93,7 +94,7 @@ describe('JobsTab', function () {
       ).toBeDefined();
     });
 
-    it('renders correct empty panel', function () {
+    xit('renders correct empty panel', function () {
```
Comment (mlunoe) [inline src/js/pages/__tests__/JobsTab-test.js]: I'm guessing this is the reason why this is WIP?

**Comment by mlunoe** [comment]
Code looks good, so it gets a label, but let's fix that test

**Comment by philipnrmn** [comment]
The hold merge is because of the missing taskdetail which will be coming soon. I'd forgotten that test, thanks for picking it up :)

**Comment by orlandohohmeier** [comment]
Unfortunately there is no header on the empty job view and the job actions are missing...

Empty Job Tree | Job Detail
--- | --- 
  | screen shot 2016-12-07 at 12 03 53

**Comment by nLight** [comment]
Action buttons are missing

master | current
--- | --- 
screen shot 2016-12-07 at 12 13 57 |  screen shot 2016-12-07 at 12 11 58

**Comment by orlandohohmeier** [comment]
**mesosphere-ci** retest this please 🙏

**Comment by orlandohohmeier** [comment]
@nLight I've addressed all issues...

**Comment by nLight** [comment]
💪  Let's get this in

**Pull request merged**